src/Sql/dbo/Stored Procedures/User_BumpAccountRevisionDateByOrganizationIds.sql

```diff
@@ -0,0 +1,18 @@
+CREATE PROCEDURE [dbo].[User_BumpAccountRevisionDateByOrganizationIds]
+    @OrganizationIds AS [dbo].[GuidIdArray] READONLY
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    UPDATE
+        U
+    SET
+        U.[AccountRevisionDate] = GETUTCDATE()
```

Comment:
I'm curious if there is another example of updating many accounts at once if we call `GETUTCDATE` once and use that value for them all or we call it live for each row. If there is an example of that and we only call it once then I'd like to switch this one to that but if there is no example or this is what the example does then this is fine.

Reply:
I did look into that, and saw we did it live for each row in both `User_BumpAccountRevisionDateByProviderUserIds` and `User_BumpAccountRevisionDateByOrganizationUserIds` so I kept with the current practice.

Reply:
Gonna mark this as resolved per
> [if] this is what the example does then this is fine.